code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -123,6 +123,8 @@
 	return ..()
 
 /datum/action/innate/cult/blood_spell/Activate()
+	if(owner.holy_check())
```

Comment:
You should probably do all these at a higher level. `/datum/action/innate/cult` or `.../cult/blood_spell`

Add a var like for the other spells if you don't want all of them to work this way.

Reply:
I did originally have the checks in attack and attack_self, but blood cult is designed where the parent calls happen after spell effects activate, not before as a check, so without a major refactor there was no way to only put this check in one place.

I wish I could have.